### PR TITLE
Apply the new pypackage cookiecutter

### PR DESCRIPTION
**Pull request by seanh**
Apply the [new pypackage cookiecutter](https://github.com/hypothesis/cookiecutters/tree/main/pypackage) to this project.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,61 +1,51 @@
...
+    format: black src tests bin
+    format: isort --atomic src tests bin
+    checkformatting: black --check src tests bin
+    checkformatting: isort --quiet --check-only src tests bin
+    lint: pylint src bin
+    lint: pylint --rcfile=tests/pyproject.toml tests
+    lint: pydocstyle src tests bin
+    lint: pycodestyle src tests bin
+    tests: coverage run -m pytest --failed-first --new-first --no-header --quiet {posargs:tests/unit/}
+    functests: pytest --failed-first --new-first --no-header --quiet {posargs:tests/functional/}
```
Comment (jon-betts) [inline tox.ini:48]: Looking at this part of the diff, putting some space between commands is easier on the eyeballs.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
+<a href="https://github.com/hypothesis/pyramid-sanity/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/workflow/status/hypothesis/pyramid-sanity/CI/main"></a>
+<a href="https://pypi.org/project/pyramid-sanity"><img src="https://img.shields.io/pypi/v/pyramid-sanity"></a>
+<a><img src="https://img.shields.io/badge/python-3.9 | 3.8-success"></a>
```
Comment (jon-betts) [inline README.md:3]: Why is this a link? Does it need to be for styling or something? Clicking on it in GH shows the badge again in a new window.

**Review by jon-betts — APPROVED** [review]
All the standard commands work, no big surprise there. We can't really review the CI etc without deploying but I can't see any reason not to.

File: README.md (hunk 2 above)
Comment (seanh) [reply README.md:3]: IIRC that was necessary to get all the badges to lay out horizontally in one row. Without the `<a>` it for some reason breaks onto the next row

Comment (seanh) [reply README.md:3]: I wonder if we can add an attr to the `<a>` to prevent it from being clickable...

Comment (seanh) [reply README.md:3]: Hmm. In the rendered HTML GitHub seems to have added an `href` to the `<a>` for us

**Pull request merged**